BParkingNano/plugins/TrackMerger.cc

```diff
@@ -154,11 +154,7 @@ void TrackMerger::produce(edm::StreamID, edm::Event &evt, edm::EventSetup const
 
     // high purity requirment applied only in packedCands
     if( iTrk < nTracks && !trk.trackHighPurity()) continue;
-   
-    // build transient track
-    const reco::TransientTrack trackTT((*(trk.bestTrack())), &(*bFieldHandle));
-    if (!trackTT.isValid()) continue;
```

Comment:
could it be the removal of this ? it's the only difference I see...

Reply:
no because when removing the sorting I have not re-introduced the isValid() check :(

Reply:
(in any case I would re-add the check)